plugins/katello/nightly/user_guide/srpm_support/index.md

```diff
@@ -0,0 +1,169 @@
+
+* The `hammer srpm info` command gathers information about a source RPM. Returning ID, Name, Version, Architecture, Epoch, Release, Filename, and Description.
+
+## Sample workflow to create a source RPM repository, upload a source RPM, and list a source RPM
+
+### Create a product and repository
+
+Since the feature is only available in the CLI/API currently, I will be using Hammer for this workflow.
+
+First we want to create a product:
+
+```bash
+# hammer product create --name Zoo --organization-id 1
+
+Product created.
+```
+
+Next we want to grab our product id for the repository creation step:
+
+```bash
+# hammer product list --organization-id 1
+
+---|------|-------------|----------------------|--------------|-----------
+ID | NAME | DESCRIPTION | ORGANIZATION         | REPOSITORIES | SYNC STATE
+---|------|-------------|----------------------|--------------|-----------
+1  | Zoo  |             | Default Organization | 0            |
+---|------|-------------|----------------------|--------------|-----------
+```
+
+Next we will create our repo:
+
+```bash
+# hammer repository create --product-id 1 --organization-id 1 --content-type yum --name source_rpms --publish-via-http yes
+
+Repository created.
+```
+
+Now we want to grab our repo id for the upload step:
+
+```bash
+# hammer repository list
+
+---|-------------|---------|--------------|----
+ID | NAME        | PRODUCT | CONTENT TYPE | URL
+---|-------------|---------|--------------|----
+1  | source_rpms | Zoo     | yum          |
+---|-------------|---------|--------------|----
+```
+
+### Upload a source RPM into a repository
+
+Now that we have our repository, we will upload a source RPM using the following command:
+
+```bash
+hammer repository upload-content --content-type srpm --id 1 --product-id 1 --path /root/garmindev-0.3.4-9.el7.src.rpm
+
+Successfully uploaded file 'garmindev-0.3.4-9.el7.src.rpm'
+```
+
+Now we will check to make sure it uploaded correctly:
+
+```bash
+# hammer repository info --id 1
+
+ID:                 1
+Name:               source_rpms
+Label:              source_rpms
+Organization:       Default Organization
+Red Hat Repository: no
+Content Type:       yum
+Mirror on Sync:     yes
+URL:
+Publish Via HTTP:   yes
+Published At:       http://centos7-katello-nightly.area51.example.com/pulp/repos/Default_Organization/Library/custom/Zoo/source_rpms/
+Relative Path:      Default_Organization/Library/custom/Zoo/source_rpms
+Download Policy:    immediate
+Product:
+    ID:   1
+    Name: Zoo
+GPG Key:
+
+Sync:
+    Status: Not Synced
+Created:            2019/08/21 15:37:14
+Updated:            2019/08/21 15:37:16
+Content Counts:
+    Packages:       0
+    Source RPMS:    1
+    Package Groups: 0
+    Errata:         0
+    Module Streams: 0
+```
+
+### Listing source RPMS
+
+To list source rpms across the entire Katello server we will run this command:
```

Comment:
Use less I/we, the same meaning is conveyed without: `To list source rpms across the entire Katello server run this command`